frontend/pages/blog/introducing-journaly.tsx

```diff
@@ -12,11 +13,11 @@ const IntroducingJournalyBlogPost: NextPage = () => (
     >
       <p>
         Hi there! I'm Robin. Long-time language enthusiast,{' '}
-        <a href="https://youtube.com/user/TheLifeOfRob">polyglot YouTuber</a>, language teacher, and
-        author of the book{' '}
-        <em>
-          <a href="https://www.kumabrand.com/shop">How To Maintain Languages</a>
-        </em>
+        <ExternalLink href="https://youtube.com/user/TheLifeOfRob">polyglot YouTuber</ExternalLink>,
+        language teacher, and author of the book{' '}
+        <ExternalLink href="https://www.kumabrand.com/shop">
+          <cite>How To Maintain Languages</cite>
```

Comment:
I did some research and learned that the [`<cite>` element](https://www.w3schools.com/tags/tag_cite.asp) is best for the title of a work! never heard of that one before...

The base styling of the `cite` element is `font-style: italic;` but that gets stripped in our `reset.css` file, so I added that italic style back to `<cite>` elements in `globalStyles.css`. Let me know what you think!